PasswordRules.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace PasswordChecker
+{
+    public class PasswordRules
+    {
+        public bool isValid = false;
+        private bool ContainsUpperLetter(string pass)
+        {
+            foreach (char c in pass)
+            {
+                if ((Char.IsLetter(c)) && (Char.IsUpper(c)))
+                    return true;
+            }
+            Console.WriteLine("Password does not contain at least one Uppercase letter");
+            return false;
+        }
+
+        private bool ContainsLowerLetter(string pass)
+        {
+            foreach (char c in pass)
+            {
+                if ((Char.IsLetter(c)) && (Char.IsLower(c)))
+                    return true;
+            }
+            Console.WriteLine("Password does not contain at least one Lowercase letter");
+            return false;
+        }
+
+        private bool ContainsDigit(string pass)
+        {
+            foreach (char c in pass)
+            {
+                if (Char.IsDigit(c))
+                    return true;
+            }
+            Console.WriteLine("Password does not contain at least one digit");
+            return false;
+        }
+
+        private bool ContainsSymbol(string pass) // 
+        {
+            string validPattern = ".!@#$%^&*";
+            foreach (char c in pass)
+            {
+                if (validPattern.Contains(c))
+                    return true;
+            }
+            Console.WriteLine("Password does not contain a special symbol");
+            return false;
+        }
+
+        private bool ContainsTenCharacters(string pass)
+        {
+            if (pass.Length >= 10)
+            {
+                return true;
+            }
+            else
+            {
+                Console.WriteLine("Password does not contain at least 10 characters");
+                return false;
+            }

```

Comment:
```suggestion
            Console.WriteLine("Password does not contain at least 10 characters");
            return false;
```

The else statement is redundant here, as `return true` will always exit the method if `pass.Length >= 10`